### DISCUSSION TITLE
Get started with Flux in Azure with multiple deployments

### DISCUSSION THREAD

**Discussion by Thaval** [fluxcd/flux2#2636] [Q&A] [answered]
Hi, 

it's a little overwhelming at the first glance. Imagine I have a git repo where are a lot of different kustomization.yaml to configure the different applications in my cluster. 
In my imagination, I want Flux to reconcile the application that was modified, for example when a deployment.yaml "a/app1/deployment.yaml" has changed. The other applications "b/app2", "c/app3", etc should not be affected.
Is the `flux create kustomize` cmd with the `path` parameter the right choice?
So I would create three flux configurations, where only the paths differ, right?

Like
```
flux create kustomization app1 \
    --source=GitRepository/contour \
    --path="./a/app1/" ...


flux create kustomization app2 \
    --source=GitRepository/contour \
    --path="./b/app2/" \

flux create kustomization app3 \
    --source=GitRepository/contour \
    --path="./c/app3/" \
```
and everything is ready?

**Comment by stefanprodan** [comment, marked as answer]
Yes you can create a Kustomization per app or per layer (apps and infra) or per tenant. See the docs and example repos: https://fluxcd.io/docs/guides/repository-structure/

**Comment by Thaval** [comment]
True, it's written there in some way. Just missed some examples of how to do it exactly.
